.github/workflows/clang-format-check.yml

```diff
@@ -1,5 +1,6 @@
 name: clang-format Check
-on: [workflow_dispatch, push, pull_request]
+on:
+  pull_request:
```

Comment:
Does this mean that clang formats will only happen when the pull request is initially opened and won't happen on changes that happen during a review?

Reply:
yeah, everything was fine on a push to my fork, but it thru an error when I created the pull-request;
------------------------------------------------------------------------------------------------------------
Committing clang-format changes
This commit does not belong to any branch on this repository.